### DISCUSSION TITLE
How to copy interface information

### DISCUSSION THREAD

**Discussion by murph-zh** [mitmproxy/mitmproxy#5268] [Q&A]
I have deployed a mitmweb in the company intranet. When other colleagues access `mitmweb>Flow>Export>Copy raw request` to copy the interface information, the copy is not successful and the clipboard is empty.

Mitmproxy: 8.0.0
Python:    3.9.1
OpenSSL:   OpenSSL 1.1.1n  15 Mar 2022
Platform:  macOS-12.1-x86_64-i386-64bit

**Comment by Prinzhorn** [comment]
https://github.com/mitmproxy/mitmproxy/issues/5264

**Comment by murph-zh** [comment]
When will this problem be fixed? @mhils @Prinzhorn 
Frustrating failure to copy interface information!

**Reply by mhils** [reply]
I realize it's frustrating for you that it doesn't work, what's even more frustrating for me is to receive negatively-worded complaints that try to pressure me into working on stuff in my free time. 🙃 

(There is no ETA, but @Pactortester's existing PR might save you very soon)
